qtpy/QtCore.py

```diff
@@ -8,18 +8,33 @@
 """
 Provides QtCore classes and functions.
 """
-
 from . import PYQT6, PYQT5, PYSIDE2, PYSIDE6, PythonQtError
 
+
 if PYQT6:
+    from PyQt6 import QtCore
```

Comment:
After running `promote_enums` below, I think we should remove `QtCore` from here, so it can't be accessed like `qtpy.QtCore.QtCore`